Chapters/4-2-Chapter24.md

```diff
@@ -0,0 +1,211 @@
+## Chapter 24
+
+<!--
+In this chapter, Pomri and Linyu set out in a small paddleboat for Heiko.
+
+They don't have fog blocking them, but they are in the dark because of the obscura.
+
+They also can't see underwater ships, like what Holrag would have. So the Holragi can spy on them and follow them to see what they're up to, maybe follow them back to Ulukoma to have a surprise attack in which they have to lock the door on Setre. The prologue illustrates a bloodied Setre, guilty for not helping Melna, paddling out to Liffa to save hae from a sinking capsule.
+
+Anyway, tangent, this chapter shows them finding Laen-Laen and Pomri's saffa, Boon-Boon, and Ginnu who can now see, but is exhausted and cold and doesn't know who thi is. However, thi knows the secret of the armory.
+
+They also need to discover Ramne's connection to Holrag
+
+They would go straight back, but they are captured by Ramne <- Too Much Capturing!
+
+Holragi is digging shelter in Heiko, because they need shelter from the coming nova that will send the Obscura to far off places.
+
+They are driven into the desert
+ -->
+
+Pomri was late, naturally, but she came prepared with a small anomin lantern in hand, and a large durable bag on her back. When she arrived, she produced a rope from the bag, tying it first around herself, then around me.
+
+"In case you fall in," she said. I remembered what it felt like to drown, and wasn't keen on doing it again.
+
+"Are you worried?" I asked as we pushed off.
+
+"About what?"
+
+"What if we find her?"
+
+Pomri paused, distracted.
+
+"That's what we're here for isn't it?"
+
+My silence agreed.
+
+There was a certain guilt I carried onto the boat. The kind of guilt that fills your belly with vines, pretangled and on fire. A guilt that made me turn around and look back to the shoreline of Ulukoma as it sank under a patchwork of ice and waves that composed the horizon. I did not long for my nather in particular. Nor for anyone back there. But the waves and the salty air made me miss that home.
```

Comment:
<3 I especially loved the first line " There was a certain guilt I carried onto the boat."